app/views/shared/_custom_phase_message_homepage.html.erb

```diff
@@ -1 +1 @@
-Complete our quick 5-question survey to <%= link_to "help us improve our content", survey_url, target: "_blank", rel: "noopener noreferrer", class: "govuk-link" %>.
+<%= link_to "Contact the Service Manual team", survey_url, class: "govuk-link" %> if you have feedback, questions or suggestions.
```

Comment:
```suggestion
<%= link_to "Contact the Service Manual team", "/service-manual/communities/contact-the-service-manual-and-service-standard-team", class: "govuk-link" %> if you have feedback, questions or suggestions.
```

As per the previous suggestion we can specify the URL here. As the user is already on www.gov.uk convention would be to not specify the absolute path.

Reply:
By the way this template is repeated in: https://github.com/alphagov/service-manual-frontend/blob/5df8cdfcca3332fee44baea1f8dba239523762fd/app/views/shared/_custom_phase_message.html.erb

which would also need editing.

Since these are identical I would suggest deleting the `_custom_phase_message_homepage.html.erb` file and editing https://github.com/alphagov/service-manual-frontend/blob/5df8cdfcca3332fee44baea1f8dba239523762fd/app/views/content_items/homepage.html.erb#L4 to `render 'shared/custom_phase_message', phase: @content_item.phase`